app/helpers/quadicon_helper.rb

```diff
@@ -70,6 +70,8 @@ def self.settings_key(klass, layout)
       when "ems_physical_infra", "ems_cloud", "ems_network", "ems_container"
         layout.to_sym
       end
+    elsif klass.base_model.to_s.underscore == "physical_switch"
+      klass.base_model.name.underscore.to_sym
     elsif klass.base_model.name != klass.name.demodulize
       klass.name.demodulize.underscore.to_sym
```

Comment:
I suggest to remove the demodulize. It was introduced by me at https://github.com/ManageIQ/manageiq-ui-classic/pull/3699/files#diff-6525cb9600835fcc2ba062fde39a2292R73 and it is causing an issue fixed at https://github.com/ManageIQ/manageiq-ui-classic/pull/3980

Removing the demodulize will help in both cases.

Reply:
@saulotoledo I removed this, now it should work fine without both conditions.